users/keppenj/index.md

```diff
@@ -4,4 +4,12 @@ layout: default
 author: Jackson Keppen
```

Comment:
Okay, thank you. I think the file is right now. I'm going to add more to it once I have my resume and a photo together.